aiida_quantumespresso/parsers/pp.py

```diff
@@ -0,0 +1,348 @@
+# -*- coding: utf-8 -*-
+""" Parser for pp.x calculations """
+from __future__ import absolute_import
+
+import traceback
+
+import numpy as np
+
+from aiida import orm
+from aiida.common import exceptions
+
+from aiida_quantumespresso.calculations.pp import PpCalculation
+from aiida_quantumespresso.utils.mapping import get_logging_container
+from .base import Parser
+
+from six.moves import range
+from six.moves import zip
+
+class PpParser(Parser):
+    """
+    Parser for pp.x calculations
+    """
+
+    # Lookup for units
+    units_dict = {
+        0: 'e/bohr^3', # Electrons, electronic charge density
+        1: 'Ry', # Total potential
+        2: 'Ry', # Ionic potential
+        3: 'states/Ry.bohr^3', # Density of states over an energy range
+        4: '', # Local density of electronic entropy - Ry/K.bohr^3?
+        5: 'states/Ry.bohr^3', # Simulated STM images from LDOS
```

Comment:
I'm not sure about this one - does STM have energy resolution, or is it just integrated over an energy window?

Reply:
Not immediately clear to me.
So in pp.x, one cannot specify an energy range. That's only possible for plot_num=3 and 10, but one can specify a `sample_bias` in Ry.

Reply:
Maybe best to remove it for now.

Reply:
Removed.